terraform/modules/department/50-aws-iam-policies.tf

```diff
@@ -749,6 +749,22 @@ data "aws_iam_policy_document" "redshift_department_read_access" {
   }
 }
 
+// MWAA Access
+
+data "aws_iam_policy_document" "mwaa_department_web_server_access" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "airflow:ListEnvironments",
+      "airflow:GetEnvironment",
```

Comment:
Just had a quick check: the permission makes sense based on this [doc ](https://docs.aws.amazon.com/mwaa/latest/userguide/access-policies.html)